### DISCUSSION TITLE
how to use  koala-1b unet and  clip text encoders

### DISCUSSION THREAD

**Discussion by CHETHAN562** [comfyanonymous/ComfyUI#2954] [Q&A] [answered]
how to use  koala-1b unet and  clip text encoders

**Comment by comfyanonymous** [comment, marked as answer]
Use the UNETLoader node for this file: https://huggingface.co/etri-vilab/koala-1b/blob/main/unet/diffusion_pytorch_model.fp16.safetensors

And the DualCLIPLoader node for these files: 
https://huggingface.co/etri-vilab/koala-1b/blob/main/text_encoder/model.fp16.safetensors
https://huggingface.co/etri-vilab/koala-1b/blob/main/text_encoder_2/model.fp16.safetensors

You can also use the CLIP from a regular SDXL checkpoint.
